etc/profile-a-l/geary.profile

```diff
@@ -47,15 +53,16 @@ include whitelist-var-common.inc
 
 apparmor
 caps.drop all
-machine-id
+ipc-namespace
```

Comment:
`ipc-namespace` may cause issues with X11.

Reply:
I commented the ipc-namespace option. Thanks!